fe1-web/model/network/method/message/data/MessageRegistry.ts

```diff
@@ -0,0 +1,35 @@
+import { ActionType, ObjectType } from './MessageData';
+import { ExtendedMessage } from '../ExtendedMessage';
+
+type HandleFunction = (msg: ExtendedMessage) => boolean;
+
+/**
+ * Represents an entry of the MessageRegistry.
+ */
+interface MessageEntry {
+  ingestion: HandleFunction; // Function to handle this type of message
```

Comment:
I'm not a big fan of the name `ingestion` here, but I don't have a better idea right now 😅